src/plopp/backends/matplotlib/utils.py

```diff
@@ -45,15 +50,18 @@ def make_figure(*args, **kwargs) -> Figure:
     F) directly.
     When using the interactive backend, we need to do more work. The ``plt.Figure``
     will not have a toolbar nor will it be interactive, as opposed to what
-    ``plt.figure`` returns. We therefore copy the minimal required code inside the
-    ``plt.figure`` function which creates a figure manager (which is apparently what
-    creates the toolbar and makes the figure interactive).
+    ``plt.figure`` returns. To fix this, we need to create a manager for the figure
+    (see https://stackoverflow.com/a/75477367).
     """
-    if not is_interactive_backend():
-        return Figure(*args, **kwargs)
-    backend = _get_backend_mod()
-    manager = backend.new_figure_manager(1, *args, FigureClass=Figure, **kwargs)
-    return manager.canvas.figure
+    fig = plt.Figure(*args, **kwargs)
+    if is_interactive_backend():
+        # Create a manager for the figure, which makes it interactive, as well as
+        # making it possible to show the figure from the terminal.
+        try:
+            plt._backend_mod.new_figure_manager_given_figure(1, fig)
+        except AttributeError:
```

Comment:
When does this happen?

Reply:
That's fine I guess :)

Reply:
This is the sequence I can reproduce, using pure MPL:
```
(base) nvaytet@this:~/code/plopp/jupyter$ ipython
Python 3.11.9 | packaged by conda-forge | (main, Apr 19 2024, 18:36:13) [GCC 12.3.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.26.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import matplotlib.pyplot as plt

In [2]: fig = plt.Figure()

In [3]: plt._backend_mod.new_figure_manager_given_figure(1, fig)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[3], line 1
----> 1 plt._backend_mod.new_figure_manager_given_figure(1, fig)

AttributeError: 'NoneType' object has no attribute 'new_figure_manager_given_figure'

In [4]: plt.show()

In [5]: plt._backend_mod.new_figure_manager_given_figure(1, fig)
Out[5]: <matplotlib.backends.backend_qt.FigureManagerQT at 0x7da165f8cbd0>
```
So until I call `plt.show()`, the `_backend_mod` seems to be `None`?

That said, I now cannot get this error to show up again when using Plopp figures.
Maybe it was related to the backend not being interactive when it should have been (i.e. the fix in `is_interactice_backend` above).
So I'll remove the `try/except` block.